recipes/barriers/build.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
+
+./autogen.sh && \
+./configure --prefix=$PREFIX --with-hash-bits=27 CFLAGS='-mcmodel=large' && \
```

Comment:
To help debugging, you can replace the `&& \` with `|| (cat config.log ; exit 1)`